src/python/tests/conftest.py

```diff
@@ -22,10 +22,10 @@
 from pathlib import Path
 
 import pytest
+from pytest import Config, FixtureRequest
 
 
 TEST_DATA_DIR_PATH = Path(__file__).parent
-ROOT_DIR_PATH = TEST_DATA_DIR_PATH.parents[2]
 FILES_DIR_PATH = TEST_DATA_DIR_PATH / "data" / "flatfiles"
```

Comment:
Why do we need the "flatfiles" subfolder?

Reply:
That is legacy and will be dropped as soon as the unit test branch is merged into `main` (not doing it now to avoid "breaking" ongoing work in other branches).

Reply:
ok. thanks!